### ISSUE TITLE
[explicit-member-accessibility] cannot be configured

### ISSUE THREAD

**Issue by viridia** [typescript-eslint/typescript-eslint#422]
Any configuration arguments beyond the first ("error") generate an error on startup.

**Repro**

`    "@typescript-eslint/explicit-member-accessibility": ["error", { "accessibility": "explicit" }],
`

```
Error: /home/talin/Projects/k4/.eslintrc.js:
        Configuration for rule "@typescript-eslint/explicit-member-accessibility" is invalid:
        Value [{"accessibility":"explicit"}] should NOT have more than 0 items.

```

**Expected Result**

Should be able to configure the rule.

**Actual Result**

Linter exits with a fatal error before processing any files.

**Additional Info**

Full configuration at https://github.com/viridia/klendathu/blob/master/.eslintrc.js

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.6.0` |
| `@typescript-eslint/parser`        | `1.6.0` |
| `TypeScript`                       | `3.3.3333` |
| `ESLint`                           | `5.16.0` |
| `node`                             | `v10.11.0` |
| `npm`                              | `6.9.0` |

**Comment by bradzacher** [comment]
We haven't released that feature to main yet.

Please use the `canary ` tag on npm: 
`npm install @typescript-eslint/eslint-plugin@canary`

**Comment by Xapphire13** [comment]
@bradzacher, if it's not released yet, why is it documented in master?

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md

**Comment by bradzacher** [comment]
We don't have a separate release branch, master is our pr target branch because it makes life easier. We release to the main tag on npm every week or two. It's a manual release so it happens when we get to it.

We have CI to automatically run a canary release for every master commit. If you want the bleeding edge always without waiting, just install from the canary tag.
So far we haven't ever had changes between canary and master, so it's safe to use.

**Comment by Xapphire13** [comment]
Ahh cool, didn't take into account what your release cycle looked like, thanks for the explanation!

**Comment by dlhart** [comment]
@bradzacher Is there a list of the rules that are currently supported in master?

**Comment by bradzacher** [comment]
@dlhart - do you mean currently supported against the `latest` tag on npm or the `canary` tag on npm?

Because the answer to your question is whatever is in master is what's supported in master......

For the code that's currently released as `latest` - we generate [git tags and release notes](https://github.com/typescript-eslint/typescript-eslint/releases) on every single `latest` release. You can use github's tag navigation to see exactly what is supported by a specific version.

The code that's currently released as `canary` is just whatever is in master.
